tests/src/Functional/PrevNextBlockTest.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Drupal\Tests\localgov_blogs\Functional;
+
+use Drupal\node\NodeInterface;
+use Drupal\Tests\BrowserTestBase;
+use Drupal\Tests\node\Traits\NodeCreationTrait;
+
+/**
+ * Tests user blocks.
+ *
+ * @group localgov_guides
+ */
+class PrevNextBlockTest extends BrowserTestBase {
+
+  use NodeCreationTrait;
+
+  /**
+   * Modules to enable.
+   *
+   * @var array
+   */
+  protected static $modules = [
+    'block',
+    'path',
+    'options',
+    'localgov_blogs',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $defaultTheme = 'stark';
+
+  /**
+   * A user with the 'administer blocks' permission.
+   *
+   * @var \Drupal\user\UserInterface
+   */
+  protected $adminUser;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp(): void {
+    parent::setUp();
+
+    $this->adminUser = $this->drupalCreateUser(['administer blocks']);
+    $this->drupalLogin($this->adminUser);
+    $this->drupalPlaceBlock('localgov_blogs_prev_next_block');
+    $this->drupalLogout();
+  }
+
+  /**
+   * Test the Previous / Next Navigation block.
+   */
+  public function testPrevNextBlock() {
+
+    // Create channel.
+    $channel = $this->createNode([
+      'title' => 'Blog Channel',
+      'type' => 'localgov_blog_channel',
+      'status' => NodeInterface::PUBLISHED,
+    ]);
```

Comment:
We're going to need to test multiple channels here, and the test case is that the blog post next / prev leads to the right post in each channel.

Reply:
Hi @andybroomfield would you be able (and have the time) to look at adding this extra testing?